archtitus.sh

```diff
@@ -35,4 +35,8 @@ echo -ne "
                     Automated Arch Linux Installer
 -------------------------------------------------------------------------
                 Done - Please Eject Install Media and Reboot
+
```

Comment:
this shouldn't require manual intervention, the script should handle this

Reply:
I don't see anywhere that the script runs mkinitcpio, which is needed in order to generate the hooks for the bootloader, and notably I don't see the one for btrfs being emitted into the mkinitcpio.conf file (however the one for LUKS is actually being emitted)

Reply:
https://github.com/ChrisTitusTech/ArchTitus/blob/main/1-setup.sh#L176

Reply:
I see the line to run it after modifying the file for LUKS. But, my point was mainly we should fix the script to do what the note says. Not make the user do it. What parameters needed to be modified for your install that you needed to run mkinitcpio again?